xbmc/addons/AddonInstaller.h

```diff
@@ -191,12 +191,13 @@ class CAddonInstallJob : public CFileOperationJob
 class CAddonUnInstallJob : public CFileOperationJob
 {
 public:
-  CAddonUnInstallJob(const ADDON::AddonPtr &addon);
+  CAddonUnInstallJob(const ADDON::AddonPtr &addon, bool removeData);
 
   virtual bool DoWork();
 
 private:
   void ClearFavourites();
 
   ADDON::AddonPtr m_addon;
+  bool m_remove; //!< True to remove add-on userdata
```

Comment:
Consider using a more descriptive name so we don't need the comment here & it's more clear in other contexts.